.github/workflows/publish-nightly.yml

```diff
@@ -31,3 +31,26 @@ jobs:
           NEXUS_USER: ${{ secrets.NEXUS_USER }}
           NEXUS_PW: ${{ secrets.NEXUS_PW }}
         run: sbt +publish
+
+      - name: Build Documentation
+        run: |-
+          sbt docs/paradox unidoc
+
+      # Create directory structure upfront since rsync does not create intermediate directories otherwise
+      - name: Create nightly directory structure
+        run: |-
+          mkdir -p target/nightly-docs/docs/pekko-connectors/${{ github.ref_name }}-snapshot/
+          mv docs/target/paradox/site/main/ target/nightly-docs/docs/pekko-connectors/${{ github.ref_name }}-snapshot/docs
+          mv target/scala-2.13/unidoc target/nightly-docs/docs/pekko-connectors/${{ github.ref_name }}-snapshot/api
```

Comment:
Maybe we can define a variable in YAML for `defaultScalaVersion`?

Reply:
The proper way to solve this issue is via an sbt plugin akin to https://github.com/lightbend/sbt-publish-rsync, there is already an issue on this https://github.com/apache/incubator-pekko/issues/104